.gitignore

```diff
@@ -59,3 +59,5 @@ typings/
 
 # other
 app/*.properties
+
+.editorconfig
```

Comment:
Why not add it?

Reply:
Have you got an example one you've added elsewhere?

Reply:
Recently added one to [simple-email-service](https://github.com/Financial-Times/simple-email-service/blob/master/.editorconfig)